lib/Client.js

```diff
@@ -130,6 +130,11 @@ class Client extends EventEmitter {
         if(defaultImageSize < 16 || defaultImageSize > 1024 || (defaultImageSize & (defaultImageSize - 1))) {
             throw new TypeError("Invalid default image size: " + defaultImageSize);
         }
+        // Set HTTP Agent on Websockets if not already set
+        if (this.options.agent && !(this.options.ws && this.options.ws.agent)) {
+            this.options.ws = this.options.ws || {};
+            this.options.ws.agent = this.options.agent;
+        }
```

Comment:
The jsdoc comment for the client constructor should be updated to reflect this new option.